tests/cross_cluster_live_migration/conftest.py

```diff
@@ -32,8 +64,478 @@ def remote_kubeconfig_export_path(request):
+    """
+    return wait_for_service_account_token(secret=remote_cluster_service_account_token_secret)
+
+
+@pytest.fixture(scope="session")
+def remote_hco_namespace(remote_admin_client):
+    return Namespace(client=remote_admin_client, name=py_config["hco_namespace"], ensure_exists=True)
+
+
+@pytest.fixture(scope="package")
+def hyperconverged_resource_scope_package_remote_cluster(remote_admin_client, remote_hco_namespace):
+    return get_hyperconverged_resource(client=remote_admin_client, hco_ns_name=remote_hco_namespace.name)
+
+
+@pytest.fixture(scope="package")
+def enabled_feature_gate_for_decentralized_live_migration_remote_cluster(
+    hyperconverged_resource_scope_package_remote_cluster,
+    remote_admin_client,
+):
+    with ResourceEditorValidateHCOReconcile(
+        patches={
+            hyperconverged_resource_scope_package_remote_cluster: {
+                "spec": {"featureGates": {"decentralizedLiveMigration": True}}
+            }
+        },
+        list_resource_reconcile=[KubeVirt],
+        wait_for_reconcile_post_update=True,
+        admin_client=remote_admin_client,
+    ):
+        yield
+
+
+@pytest.fixture(scope="package")
+def enabled_feature_gate_for_decentralized_live_migration_local_cluster(
+    hyperconverged_resource_scope_package,
+    admin_client,
+):
+    with ResourceEditorValidateHCOReconcile(
+        patches={
+            hyperconverged_resource_scope_package: {"spec": {"featureGates": {"decentralizedLiveMigration": True}}}
+        },
+        list_resource_reconcile=[KubeVirt],
+        wait_for_reconcile_post_update=True,
+        admin_client=admin_client,
+    ):
+        yield
+
+
+@pytest.fixture(scope="package")
+def network_for_live_migration_local_cluster(admin_client, hco_namespace):
+    return NetworkAttachmentDefinition(
+        name=LIVE_MIGRATION_NETWORK_NAME, namespace=hco_namespace.name, client=admin_client, ensure_exists=True
+    )
+
+
+@pytest.fixture(scope="package")
+def network_for_live_migration_remote_cluster(remote_admin_client, remote_hco_namespace):
+    return NetworkAttachmentDefinition(
+        name=LIVE_MIGRATION_NETWORK_NAME,
+        namespace=remote_hco_namespace.name,
+        client=remote_admin_client,
+        ensure_exists=True,
+    )
+
+
+@pytest.fixture(scope="package")
+def configured_hco_live_migration_network_local_cluster(
+    hyperconverged_resource_scope_package,
+    admin_client,
+    network_for_live_migration_local_cluster,
+):
+    """
+    Configure the live migration network for HyperConverged resource.
+    """
+    with ResourceEditorValidateHCOReconcile(
+        patches={
+            hyperconverged_resource_scope_package: {
+                "spec": {"liveMigrationConfig": {"network": network_for_live_migration_local_cluster.name}}
+            }
+        },
+        list_resource_reconcile=[KubeVirt],
+        wait_for_reconcile_post_update=True,
+        admin_client=admin_client,
```

Comment:
iirc this one will also cause virt-handler pods to be re-created (with new network spec)
should you also wait for them to reconcile here?